sdk/ai/ai-inference-rest/test/public/embeddings.spec.ts

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { createRecorder, createEmbeddingsClient } from "./utils/recordedClient.js";
+import { Recorder } from "@azure-tools/test-recorder";
+import { assert, beforeEach, afterEach, it, describe } from "vitest";
+import { ModelClient, isUnexpected, EmbeddingsResultOutput } from "../../src/index.js";
+
+describe("embeddings test suite", () => {
```

Comment:
should we add tests for image embedding?

Reply:
v3, given the ASAP request